config/environments/development.rb

```diff
@@ -81,12 +81,13 @@
   # Uncomment if you wish to allow Action Cable access from any origin.
   # config.action_cable.disable_request_forgery_protection = true
 
-  # Uncomment if you wish to use codespaces
-  # config.action_controller.forgery_protection_origin_check = false
-
   config.hosts << ENV["DEV_HOSTS"]
   config.hosts << ".app.github.dev"
 
+  if ENV["CODESPACES"] === "true"
+    config.action_controller.forgery_protection_origin_check = false
+  end
+
```

Comment:
I don't love this but I could not find a better solution.

Reply:
I like the env var protection